dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/DeletedUserHelper.java

```diff
@@ -0,0 +1,30 @@
+package io.dockstore.webservice.helpers;
+
+import java.sql.Timestamp;
+import java.time.Duration;
+import java.time.LocalDateTime;
+
+import io.dockstore.webservice.core.DeletedUsername;
+import io.dockstore.webservice.jdbi.DeletedUsernameDAO;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public final class DeletedUserHelper {
+    // NIH suggests a 3 year limit before allowing username reuse.
+    private static final Duration NO_REUSE_TIME_LIMIT = Duration.ofDays(355 * 3);
```

Comment:
365.

Reply:
Some random guy said Period is more suitable https://stackoverflow.com/a/29899551  . Might be worth looking into

Reply:
Neat